www/front_src/src/Resources/Details/tabs/Details/DetailsCard/cards.tsx

```diff
@@ -217,6 +218,23 @@ const getDetailCardLines = ({
         },
       ],
     },
+    {
+      title: labelGroups,
+      field: details.groups,
+      xs: 12,
+      getLines: (): Lines => [
+        {
+          key: 'groups',
+          line: (
+            <div>
```

Comment:
Maybe it's here to add class or style CSS for margin between chips

Reply:
I think, it's better to override the `className` on the `Chip` component below

Reply:
I think, it's better to override the `className` on the `Chip` component below